docs/source/best-practices/homequota.rst

```diff
@@ -0,0 +1,63 @@
+*******************************************************
+Stay Under your Quota at /home/
+*******************************************************
+It is vital to stay under your ``/home/<username>`` quota to prevent issues using Open OnDemand and performing tasks on Discovery. Here are some suggestions to stay under quota and general best practices with using your ``/home/<username>`` directory.
+
+Utlilize /scratch and /work Directories
```

Comment:
Why not just "/scratch and /work"?

Reply:
I agree this could be rephrased slightly. I think 'Utilizing /scratch and /work' would work and we can drop the directories from the title. I would want to include 'Utilize' just because the page is about staying under your $HOME quota and it might be strange to have a heading of /work and /scratch. I am totally open to change it to just that if others are in agreement.